can-io-firmware/send_pwm.py

```diff
@@ -7,23 +7,21 @@
 import uavcan
 import os
 
-DSDL_DIR = os.path.join(os.path.dirname(__file__), '../uavcan_data_types/cvra')
+DSDL_DIR = os.path.join(os.path.dirname(__file__), "../uavcan_data_types/cvra")
```

Comment:
Why double quote when you can single quote?

Reply:
We just need to pick one and stick to it. So far it is mixed, even in a single file.